cdap-log-appender-spi/src/main/java/io/cdap/cdap/logs/ILogAppender.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.logs;
+
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import java.io.Closeable;
+
+public interface ILogAppender extends Closeable {
```

Comment:
Are there any specific reason to introduce a new SPI interface instead of using ch.qos.logback.core.Appender ?

Reply:
As discussed offline renamed it to LogPublisher.